bin/install.sh

```diff
@@ -45,6 +45,12 @@ install_args() {
     fi
 }
 
+install_assert_centos() {
+    if [[ $install_os_release_id != almalinux ]]; then
```

Comment:
This needs to be almalinux|centos. `install_assert_os_centos`

Also this should be two functions: install_os_is_centos and install_os_assert_centos. I find the latter probably is less useful than the former so could just be one function, and the clients just call install_err